db/migrate/20230303002215_change_verified_boolean_to_date.rb

```diff
@@ -0,0 +1,8 @@
+class ChangeVerifiedBooleanToDate < ActiveRecord::Migration[7.0]
+  def change
+    %w{home code docs wiki mail bugs}.each do |name|
```

Comment:
Would it be better to have a verified_links column that can hold a list of the links instead of booleans? 

Pro: This would allow for more flexibility if we add/change link types in the future. 
Con: Maybe that's overkill since we have a separate column for each link type on linksets so we'd be making db changes anyway 🤷‍♀️

Reply:
We switched to holding `verified_at` timestamps rather than booleans. Would this still make sense if it's not booleans?

The Linkset model does feel denormalized. Maybe that's an optimization but it also makes the model odd to interact with.

Reply:
Ah you're right. Thank you. I think sticking with the pattern makes sense here. I'm assuming it's denormalized for a reason.